src/Framework/MetaProjectGeneratedEventArgs.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Build.Framework
+{
+    public class MetaProjectGeneratedEventArgs : BuildMessageEventArgs
```

Comment:
Left this as a skeleton since I'm looking for feedback if there's an existing EventArgs class I should be using for this instead of creating a new one.

Reply:
I think creating a new one is reasonable.